src/app/file-manager/services/gcs-api.service.ts

```diff
@@ -24,13 +34,71 @@ export class GcsApiService extends GcsService {
   }
 
   public uploadFiles(bucketName: String, files: any[]) {
-      if (files !== null && files.length > 0) {
-        this.electronService.ipcRenderer.send('start-upload', bucketName, files, SecurityService.getAccessToken());
-      }
+    if (files !== null && files.length > 0) {
+      this.electronService.ipcRenderer.send('start-upload', bucketName, files, SecurityService.getAccessToken());
+    }
   }
 
   public downloadFiles(files: Item[]) {
     this.electronService.ipcRenderer.send('start-download', files, SecurityService.getAccessToken());
   }
 
+  public cancelAll() {
```

Comment:
As far as I can tell these are not GCS related operations, so they don't belong in this class.

Reply:
If it's ok for you this will be moved to a new service in a next PR.

Reply:
OK